homeassistant/components/hive/climate.py

```diff
@@ -69,9 +80,13 @@ def supported_features(self):
     @property
     def name(self):
         """Return the name of the Climate device."""
-        friendly_name = "Heating"
-        if self.node_name is not None:
-            friendly_name = f"{self.node_name} {friendly_name}"
+        if self.device_type == "Heating":
+            friendly_name = "Heating"
+            if self.node_name is not None:
+                friendly_name = "{} {}".format(self.node_name, friendly_name)
+        elif self.device_type == "TRV":
+            friendly_name = self.node_name
+
         return friendly_name
```

Comment:
If none of the cases above are true, we'll error here. Please make sure to define `friendly_name` first regardless.

Reply:
@MagicalTrev89, this should go back to the way it was. That way we won't break home assistant if a user names devices the same within the hive app.